x-pack/plugins/canvas/canvas_plugin_src/functions/common/sort.ts

```diff
@@ -28,12 +28,14 @@ export function sort(): ContextFunction<'sort', Datatable, Arguments, Datatable>
         types: ['string'],
         aliases: ['_', 'column'],
         multi: false, // TODO: No reason you couldn't.
+        required: true,
```

Comment:
I don't think this should be required. It'll use the first column by default, and the help text says as much too:

> 'The column to sort on. If column is not specified, the {datatable} will be sorted on the first column.'

Reply:
Right, I missed that. I reverted the changed